### DISCUSSION TITLE
tailwind plugins

### DISCUSSION THREAD

**Discussion by AyesanAmH** [tailwindlabs/tailwindcss#6002] [Help] [answered]
Tailwind  plugins  including line-clamp and typography 
not working.Why?

**Comment by simonswiss** [comment, marked as answer]
Hey!

Do you have a code example that shows the plugins not working? What exactly is not working? What version of Tailwind are you using, and what version of the plugins?

Ideally, you'd need to share a little code example (Tailwind Play or GitHub repo) that demonstrates what's not working, in order to get help.

Happy to have a look at it if you share something 👍

**Reply by AyesanAmH** [reply]
Thank you for your help.
I've now fixed it. It is because of tailwind installation.
not plugins error.
